how-to/install/multimc.md

```diff
@@ -0,0 +1,67 @@
+---
+hidden: true
+---
+
+# MultiMC
+
+[MultiMC](https://multimc.org/) does not support automatic updates by default. However, FO has developed a custom MultiMC pack to perform updates automatically. Read more about the [MultiMC auto-updating pack](multimc.md#how-does-it-work)
+
+{% hint style="info" %}
+You may also want to use one of the other [launchers supported by FO](./), which offer easier installation, updates and mod management.
+{% endhint %}
+
+{% tabs %}
+{% tab title="Automatic Updates (recommended)" %}
+1. Download and install [Java 21](https://download.fo/java21)
```

Comment:
I suggest adding an info box above saying "While this variant allows for automatic updates within a specific Minecraft version, it is more difficult to update across Minecraft versions and to disable built-in mods if necessary. Consider using Prism Launcher instead, which also has a version updater."